test/unit/edgeArrayUnitTest.js

```diff
@@ -0,0 +1,40 @@
+var assert = require('assert');
+var ag = require('../../lib');
+var agens = require('../../lib/agens.js');
+var g = require('./testFunctions.js')
+
+describe('VertexUnitTest suite', function() {
+
+    var edgeArray = agens.parse('[e1[6.6][7.9,8.3]{"s1":"a"},e2[5.7][2.3,4.8]{"s2":"b"},e3[9.3][3.2,4.4]{"s3":"c"}]', {startRule: 'EdgeArray'});
```

Comment:
Do attributes need quotes?

Reply:
Fixed it, thanks for pointing that out :+1: